setup.py

```diff
@@ -37,7 +37,10 @@ def run(self):
     "nd2": ["nd2[legacy]>=0.2.0"],
     "dv": ["mrc>=0.2.0"],
     "bfio": ["bfio>=2.3.0", "tifffile<2022.4.22"],
-    # "czi": ["aicspylibczi>=3.0.5"],  # excluded for licensing reasons
+    "czi": [
+        "fsspec>=2022.7.1",
+        # "aicspylibczi>=3.0.5",  # excluded for licensing reasons
+    ],
```

Comment:
this line is giving me pause.  You can install `[czi]` and not get czi support?  I'm not sure if this is helping.